wagtail/wagtailadmin/static/wagtailadmin/css/.gitignore

```diff
@@ -0,0 +1,9 @@
+core.css
+layouts/home.css
+layouts/login.css
+layouts/page-editor.css
+layouts/preview.css
+normalize.css
+panels/rich-text.css
+userbar.css
+userbar_embed.css
```

Comment:
Sounds good to me. Keeping track of which .css files are and aren't autogenerated seems like it could be a headache otherwise.